### PR TITLE
Further enhancements to #1558

### PR DESCRIPTION
**Pull request by okurz**

### CODE REVIEW

File: autotest.pm [hunk 1]
```diff
@@ -112,11 +112,28 @@ sub loadtest {
...
+        # Adding the include path of os-autoinst into python context
+        my $inc = File::Basename::dirname(__FILE__);
+        $code .= "
+            use base 'basetest';
+            use Mojo::File 'path';
+            use Inline Python => 'import sys; sys.path.append(\"$inc\")';
+	    #use testapi;
+	    #use mmapi;
+	    #use Inline Python => 'for i in dir(perl.testapi): locals()[i] = getattr(perl.testapi, i)';
+	    #use Inline Python => 'for i in dir(perl): for j in i: locals()[j] = getattr(i, j)';
```
Comment (Martchus) [inline autotest.pm:129]: I suppose these are still a leftover.

File: t/data/tests/tests/pre_boot.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
+# Copyright (C) 2019 SUSE LLC
```
Comment (Martchus) [inline t/data/tests/tests/pre_boot.py:1]: This PR has really lying around for a while now; you could update the year. The same counts for the other occurrences.

File: t/data/tests/tests/pre_boot.py [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+import sys
+print(sys.path)
+from testapi import *
+
+def run(self):
+    send_key('esc')
+    if not check_screen('should_not_match', 0):
+        return
+    raise Exception('Should not reach here')
```
Comment (Martchus) [inline t/data/tests/tests/pre_boot.py:24]: Out of curiosity: What would happen if we'd actually thrown an exception from Python? Is the error shown appropriately?

Comment (okurz) [reply t/data/tests/tests/pre_boot.py:24]: yes, the perl code will invoke the customized die-handler

File: t/data/tests/tests/pre_boot.py (hunk 2 above)
Comment (okurz) [reply t/data/tests/tests/pre_boot.py]: done

File: autotest.pm (hunk 1 above)
Comment (okurz) [reply autotest.pm]: done

**Comment by Martchus** [comment]
I only haven't approved because it looks like #1558 should be merged first.

**Comment by okurz** [comment]
correct. I created this separate PR as it shows with CI checks that everything is ok whereas in https://github.com/os-autoinst/os-autoinst/pull/1558 we have custom OBS checks due to container image builds in that temporary OBS project.

**Comment by okurz** [comment]
https://github.com/os-autoinst/os-autoinst/pull/1558 is now merged. I rebased to fixed the conflict.

**Review by Martchus — APPROVED** [review]

**Pull request merged**